plugin/jetpack.vim

```diff
@@ -683,6 +687,16 @@ function! s:available_packages() abort
   return s:available_packages
 endfunction
 
+function! s:ask(message)
+  call inputsave()
+  echohl WarningMsg
+  let answer = input(a:message)
+  echohl None
+  call inputrestore()
+  echo "\r"
+  return answer =~? '^y'
+endfunction
```

Comment:
This is a copy of vim-plug. It's licence is MIT.
https://github.com/junegunn/vim-plug/blob/master/plug.vim#L316